### PR TITLE
.

### PR DESCRIPTION
**Pull request by Polka228**

### CODE REVIEW

File: phone/main .go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"text/scanner"
+)
+
+func createPhoneBook(names []string, numbers []string) map[string]string {
+	m := make(map[string]string)
+	file := bufio.Reader(phone_numbers.txt) //??
```
Comment (zayac) [inline phone/main .go:10]: You should have been reading the file in `main()`.

File: phone/test.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestCreatePhoneBook(t *testing.T) {
```
Comment (zayac) [inline phone/test.go:8]: Can you add a test containing duplicate names/phone numbers?

**Review by zayac — COMMENTED** [review]
The program contains compilation errors. Can you fix them?
